src/api/ens.js

```diff
@@ -180,6 +206,8 @@ export {
   getNamehashWithLabelHash,
   getResolverContract,
   getResolverReadContract,
+  getOldResolverContract,
```

Comment:
This feels a bit wrong to me. If we have another new resolver this will get messy. Do we want to take some argument to know which Resolver Contract it is?

Reply:
I would suggest doing the refactoring when another resolver needs to be supported as it may come with different requirement.